src/Tools/dotnet-trace/CommandLine/Commands/PrintLiveCommandHandler.cs

```diff
@@ -0,0 +1,72 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using Microsoft.Diagnostics.Tracing;
+using Microsoft.Diagnostics.Tracing.Parsers;
+using System;
+using System.CommandLine;
+using System.CommandLine.Binding;
+using System.Threading;
+using System.Threading.Tasks;
+
+namespace Microsoft.Diagnostics.Tools.Trace
+{
+    internal static class PrintLiveCommandHandler
+    {
+        delegate Task<int> PrintLiveDelegate(CancellationToken ct, IConsole console, int processId, uint buffersize, string providers, string profile, TimeSpan duration);
+
+        /// <summary>
+        /// Collects a diagnostic trace from a currently running process.
+        /// </summary>
+        /// <param name="ct">The cancellation token</param>
+        /// <param name="console"></param>
+        /// <param name="processId">The process to collect the trace from.</param>
+        /// <param name="buffersize">Sets the size of the in-memory circular buffer in megabytes.</param>
+        /// <param name="providers">A list of EventPipe providers to be enabled. This is in the form 'Provider[,Provider]', where Provider is in the form: 'KnownProviderName[:Flags[:Level][:KeyValueArgs]]', and KeyValueArgs is in the form: '[key1=value1][;key2=value2]'</param>
+        /// <param name="profile">A named pre-defined set of provider configurations that allows common tracing scenarios to be specified succinctly.</param>
+        /// <returns></returns>
+        private static Task<int> PrintLive(CancellationToken ct, IConsole console, int processId, uint buffersize, string providers, string profile, TimeSpan duration)
```

Comment:
Can we call this command `monitor` instead of `print-live`? `dotnet-counters` already supports two mechanisms - live streaming the data onto console (which is what this command does), and dumping the data to a file (which is what `dotnet-trace collect` does). `dotnet-counters` use `monitor` and `collect`, so it would be nice to use the same command sets for the two tools rather than creating a whole new keyword.

Reply:
Yea, `monitor` is a lot better.